src/java/org/apache/cassandra/service/accord/db/AccordData.java

```diff
@@ -120,11 +121,18 @@ FilteredPartition get(PartitionKey key)
         return getDeserialized(key);
     }
 
+    @Override
+    public Iterator<FilteredPartition> iterator()
+    {
+        return serialized.values().stream().map(this::deserialize).iterator();
+    }
+
     @Override
     public Data merge(Data data)
     {
         AccordData that = (AccordData) data;
         AccordData merged = new AccordData();
+        //TODO on conflict should we "merge" the partition rather than override?
```

Comment:
a conflict would mean datasets on replicas have diverged, which would mean a pretty serious accord bug. I think we'd want to throw an exception in that case and report it to the agent. We should probably add a checked exception to the method in accord and handle it there